--- WebServer.cpp
+++ WebServer.cpp
@@ -321,5 +321,5 @@
-//           if (_currentClient.connected()) {
-//             _currentStatus = HC_WAIT_CLOSE;
-//             _statusChange = millis();
-//             keepCurrentClient = true;
-//           }
+           if (_currentClient.connected()) {
+             _currentStatus = HC_WAIT_CLOSE;
+             _statusChange = millis();
+             keepCurrentClient = true;
+           }
